src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php

```diff
@@ -144,16 +144,17 @@ public function load(array $configs, ContainerBuilder $container): void
         $configuration = $this->getConfiguration($configs, $container);
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setParameter($this->getAlias() . '.name', $config['name']);
```

Comment:
removed the `getAlias()` and used hardcoded value to make the parameters easier to find with searches